pilot/platform/cloudfoundry/servicediscovery.go

```diff
@@ -71,15 +71,14 @@ func newService(hostname string) *model.Service {
 		Ports: []*model.Port{
 			{
 				Port:     AppPort,
-				Protocol: model.ProtocolTCP,
+				Protocol: model.ProtocolHTTP,
```

Comment:
is everything HTTP? isn't it possible to expose TCP services via CF apps?

Reply:
CF apps can expose TCP services, but for the near term we will be focusing on HTTP. In the future, I do not think it should be hardcoded in the platform adapter one way or the other.